scripts/globals/spells/bluemagic/uppercut.lua

```diff
@@ -24,25 +24,26 @@ end
 function onSpellCast(caster,target,spell)
     local params = {}
     -- This data should match information on http://wiki.ffxiclopedia.org/wiki/Calculating_Blue_Magic_Damage
-        params.tpmod = TPMOD_ATTACK
-        params.damageType = tpz.damageType.HTH
-        params.scattr = SC_LIQUEFACTION
-        params.scattr2 = SC_IMPACTION
-        params.numhits = 1
-        params.multiplier = 1.5
-        params.tp150 = 1.5
-        params.tp300 = 1.5
-        params.azuretp = 1.5
-        params.duppercap = 39
-        params.str_wsc = 0.35
-        params.dex_wsc = 0.0
-        params.vit_wsc = 0.0
-        params.agi_wsc = 0.0
-        params.int_wsc = 0.0
-        params.mnd_wsc = 0.0
-        params.chr_wsc = 0.0
+    params.tpmod = TPMOD_ATTACK
+    params.attackType = tpz.attackType.PHYSICAL
+    params.damageType = tpz.damageType.HTH
```

Comment:
And for example, here we have it as "hand-to-hand" when bgwiki lists it as "blunt"

(I know you weren't responsible for this, mrhappyasthma)

Reply:
This should be blunt, it seems. From the various websites that have it documented as such.

I can't find any reference to H2H

Reply:
Done.